migrations/20190916145342_add_user_alias.js

```diff
@@ -0,0 +1,10 @@
+
+exports.up = function (knex) {
+  return knex.schema.table("user", function (table) {
+    table.string("alias").defaultTo("");
```

Comment:
I just realized we should default this to bill so the schema change is non-locking